### ISSUE TITLE
Load LUA scripts at startup

### ISSUE THREAD

**Issue by xue-cheng** [dail8859/NotepadNext#331]
I have noticed that `editor.StyleFont[0] = ...` can be used to change the editor's font. However, I have yet to find a way to save changes permanently. Since an embedded LUA console already exists, it should be a workaround that loads a script file at each startup.

**Comment by dail8859** [comment]
> However, I have yet to find a way to save changes permanently

Correct. Unfortunately everything is hard coded and not possible to change it permanently.

> it should be a workaround that loads a script file at each startup.

Something like this could possibly work. There's have to be a hook to set the editor font every time a new editor is created or a new lexer is used. It simply can't just be ran once.
